.travis.yml

```diff
@@ -5,6 +5,7 @@ php:
     - 7.2
     - 7.3
     - 7.4
+    - nightly
```

Comment:
Remove this nightly. This will run a nightly build in addition to the job below

Reply:
Ok, not sure why we don't have 2 jobs on doctrine/annotations then

Reply:
Because I removed it there

Reply:
Ah that makes sense :joy: